docusaurus/docs/iOS/combine/combine-overview.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Overview
+---
+
+The `StreamChat` framework ships with controllers that are fully compatible with `Combine`.
+You can use these combine-compatible controllers to observe the changes to the controllers, and build your own views to consume the data.
+
+Each controller exposes publishers based on the kind of data they control and allow you to observe changes. You can find examples on how to build your own views using these publishers here.
+
+## Using the `Combine` extensions of Controllers
+
+When using combine extensions of the controllers, you need to call the `synchronize` method on your controller instance before starting to observe the publishers.
```

Comment:
Are you sure that this is needed for `Combine` publishers as well?

Reply:
`Combine` publishers internally use the delegates, and I confirmed that `synchronize` is needed when using them too.